jobs/business-sync/src/business_sync/job.py

```diff
@@ -35,6 +38,18 @@ def create_app(run_mode=os.getenv("FLASK_ENV", "production")):
     return app
 
 
+def start_cloud_sql_proxy(app):
+    """Start a cloud sql proxy"""
+    cmd = [
+        "cloud-sql-proxy",
+        f"--credentials-file={app.config['WAREHOUSE_CREDENTIALS_FILE']}",
```

Comment:
are you sure this will work? does --credentials-file takes a file path only, or can it take file contents? if it is the path, you would need to mount it in the container

Reply:
It is working locally, but I had the credentials file in root directory for local, not sure if it's gonna work the same for GCP?

Reply:
in .env it is a file path

Reply:
and in 1password we store contents of WAREHOUSE_CREDENTIALS_FILE. to test if it works you can replace value in .env with contents. if it does not work locally, it won't local anywhere (gcp included). then you to modify logic in your config.py to create and mount the file with contents from WAREHOUSE_CREDENTIALS_FILE in the container